test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9069,13 +9069,15 @@ def forward(self, boxes, size):
                       dynamic_axes={"size": [0, 1]},
                       test_with_inputs=[(boxes, size), (boxes, size_2)])
 
+    @skipIfUnsupportedMaxOpsetVersion(16)  # TODO: Opset 16 RoiAlign result mismatch
```

Comment:
```suggestion
    @skipIfUnsupportedMaxOpsetVersion(15)  # TODO: Opset 16 RoiAlign result mismatch
```
my bad, this is inclusive.

Reply:
pop, this is failing in CI still ..

Reply:
oops, mistakenly reverted the change when rebasing. 06f81649259289f7779cb14d3ecfaafa6b1f33d6 should have done this.